htdocs/lang/de/ocstyle/main.tpl.php

```diff
@@ -133,6 +133,18 @@ function flashbutton(bname)
                     document.getElementsByName(bname)[0].className = "formbutton_active";
                     window.setTimeout('resetbutton(\'' + bname + '\')', 350);
                 }
+
+                function toggleMenu(divid) {
```

Comment:
Also here I would like if we use one way of writing names which is already used.